lib/util/noop.js

```diff
@@ -1,6 +1,6 @@
 "use strict";
 
-var noop = function() {};
+var noop = Function.prototype;
```

Comment:
interesting approach. i was looking for more info on this idea on the internet and stumbled on this jsperf http://jsperf.com/noop-vs-function-prototype. Weirdly, I see a wide variation in performance of Function.prototype as noop depending on browser. In my Safari (8.0.8) it was extremely slow compared to `function() {}`. In chrome (44) it was among the fastest. In firefox (37), it was far and away the fastest (as i wrote this, firefox wants to update to 40 haha).